mock/service/node.go

```diff
@@ -51,7 +51,11 @@ func (s *service) NodeStageVolume(
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 	if !exists {
-		status.Errorf(codes.Internal, "staging target path %s does not exist", req.StagingTargetPath)
+		// If target path does not exist we need to create the directory where volume will be staged
+		if err = os.MkdirAll(req.StagingTargetPath, os.FileMode(0755)); err != nil {
```

Comment:
According to the csi spec, the CO is responsible for creating the staging path. So we should not mkdir for nodestage.

Reply:
I've returned this back to the original code.